torchvision/models/vision_transformer.py

```diff
@@ -313,6 +322,15 @@ def _vision_transformer(
     "interpolation": InterpolationMode.BILINEAR,
 }
 
+_COMMON_SWAG_META = {
+    "task": "image_classification",
+    "architecture": "ViT",
+    "publication_year": 2022,
+    "recipe": "https://github.com/facebookresearch/SWAG",
+    "categories": _IMAGENET_CATEGORIES,
+    "interpolation": InterpolationMode.BICUBIC,
```

Comment:
Nit: Should we share some of the parameters from common and overwrite them here?

Something like:
```python
_COMMON_SWAG_META = {
    **_COMMON_META,
    "publication_year": 2022,
    "recipe": "https://github.com/facebookresearch/SWAG",
 }
```

We also need to confirm what the interpolation that they used is.

Reply:
oh, didn't know we can overwrite like that! Good point!

For the interpolation, I got it from their eval method: https://github.com/facebookresearch/SWAG/blob/main/imagenet_1k_eval.py#L198

and to me it seems like they are using `BICUBIC`.